parsers/PH.py

```diff
@@ -0,0 +1,696 @@
+    # Handle storage
+    if "production.hydro_storage" in df.columns:
+        df = df.rename(columns={"production.hydro_storage": "storage.hydro"})
+        df["storage.hydro"] *= -1
+    return df
+
+
+def download_exchange_market_reports_items(
+    session: Session, reports_items: List[MarketReportsItem], logger: Logger
+) -> pd.DataFrame:
+    _all_items_df = []
+    for reports_item in reports_items:
+        logger.debug(f"Downloading market reports for {reports_item.filename}")
+        r: Response = session.get(reports_item.link)
+        _df = pd.read_csv(r.url)
+        _df = _df[:-1]
+        _df = convert_column_to_datetime(_df, "RUN_TIME")
+        _df["zone_key"] = _df["HVDC_NAME"].apply(
+            lambda x: EXCHANGE_KEY_MAPPING[x]["zone_key"]
+        )
+        _df["net_flow"] = _df.apply(
+            lambda x: EXCHANGE_KEY_MAPPING[x["HVDC_NAME"]]["flow"] * x["FLOW_FROM"],
+            axis=1,
+        )
+        _df = _df[["datetime", "zone_key", "net_flow"]]
+        _all_items_df.append(_df)
+
+    df = pd.concat(_all_items_df, ignore_index=True)
+    return df
+
+
+def convert_column_to_datetime(df: pd.DataFrame, datetime_column: str) -> pd.DataFrame:
+    """convert datetime column to datetime format"""
+    dt1 = pd.to_datetime(
+        df[datetime_column],
+        format="%m/%d/%Y %I:%M:%S %p",
+        errors="coerce",
+    )
+    dt2 = pd.to_datetime(df[datetime_column], format="%m/%d/%Y", errors="coerce")
+    df["datetime"] = dt1.combine_first(dt2)
+    return df
+
+
+@refetch_frequency(timedelta(days=1))
+def fetch_production(
+    zone_key: ZoneKey = "PH-LU",
+    session: Session = Session(),
+    target_datetime: Optional[datetime] = None,
+    logger: Logger = getLogger(__name__),
+) -> list:
+    reports_items = get_all_market_reports_items("production", logger)
+    reports_items = filter_reports_items(reports_items, target_datetime)
+
+    if len(reports_items) == 0:
+        raise ParserException(
+            parser="PH.py",
+            zone_key=zone_key,
+            message=f"{zone_key}: No production data available for {target_datetime.date()} ",
+        )
+
+    df = download_production_market_reports_items(session, reports_items, logger)
+    df = (
+        df.pipe(filter_for_zone, zone_key)
+        .pipe(filter_generation)
+        .pipe(match_resources_to_modes, logger)
+        .pipe(aggregate_per_datetime_mode)
+        .pipe(pivot_per_mode)
+    )
+
+    production_breakdown = ProductionBreakdownList(logger)
+    for datetime, row in df.iterrows():
+        production_mix = ProductionMix()
+        for mode in [m for m in row.index if "production." in m]:
+            production_mix.add_value(mode.replace("production.", ""), row[mode])
+        storage_mix = StorageMix()
+        if "storage.hydro" in row.index:
+            storage_mix.add_value("hydro", row["storage.hydro"])
+        production_breakdown.append(
+            zone_key,
+            datetime,
```

Comment:
This is not returned as a native datetime, we need to fix that.
The following works (if we rename the df datetime so there are no conflicts) but I'm not sure if it's the best way to go.
```suggestion
            datetime.fromisoformat(date_time.__str__()),
```